app/assets/stylesheets/application.scss

```diff
@@ -16,6 +16,15 @@
 @import "bootstrap-sprockets";
 @import "bootstrap";
 
+.flash-message {
+  color: green;
+  font-size: 25px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  margin-left: -200px;
+  margin-top: -25px;
+}
```

Comment:
I love the color change and the centering of this flashmessage. I thing we just have to change the error flash messages to be a little different.